### DISCUSSION TITLE
Slow Upload - 278KB/s

### DISCUSSION THREAD

**Discussion by krezlic** [deepgram/community#800] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

Doing tests on just the API playground and the upload speeds are woeful, only getting 278KB/s on upload of a WAV audio file.
I have seen it much faster in earlier testing - Is this something specific to my testing or something can resolve?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
hi @krezlic 

what region are you in?

**Comment by abhay-111** [comment]
@team-deepgram I am also facing the same Issue
`{"message":"{\"err_code\":\"SLOW_UPLOAD\",\"err_msg\":\"Request upload timeout.\",\"request_id\":\"7d4ea589-209c-4b98-a553-081a1885c678\"}","name":"DeepgramApiError","status":408}`
